tech/languages/java/java-build-tools-installation.md

```diff
@@ -24,22 +24,46 @@ Last but not least, still quite popular duo [Ant+Ivy](http://ant.apache.org/ivy/
 $ sudo dnf install ant apache-ivy
 ```
 
-## DevAssistant is obsolete, use it for reference only
+## Creating a Maven project
 
-You can get started with Maven by using [DevAssistant](/tools/devassistant/about.html) to create a simple project, simply type (after you have DevAssistant setup and installed):   
+### Using `mvn` command-line tool
+
+Use the command from the [Maven quick start guide](https://maven.apache.org/guides/getting-started/maven-in-five-minutes.html):
+
+```
+$ mvn archetype:generate -DgroupId=com.mycompany.app -DartifactId=my-app -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4 -DinteractiveMode=false
+```
+
+The project that is created follows the [Maven Project Structure](http://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html). Initially you should see the `src` directory when you explore the project directory. You will also notice that your project directory has a file, the [pom.xml](https://maven.apache.org/pom.html) in it, this is your Maven project file that defines your build, and how you will transition through the Maven build [lifecycles](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html). 
+
+The sample application created by the Maven archetype contains a single "Hello World" application (jar).
+
+The `target` directory will be created after the first compilation: run 
 
 ```
-da --debug crt java maven -n Dev_Sample
+$ mvn compile
 ```
 
-The project that is created (Example: Dev_Sample), roughly follows the [Maven Project Structure](http://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html) but does not contain all of the directories or files. Initially you should see 2 directories (`src` and `target`) when you explore the project directory. You will also notice that your project directory has a file, the [pom.xml](https://maven.apache.org/pom.html) in it, this is your Maven project file that defines your build, and how you will transition through the Maven build [lifecycles](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html). 
+This directory is where your compiled artifacts, or generated outputs (from builds) are stored. Its best to think of this directory as a temp space, used to store the outputs from a build. 
+
+### Creating a git repo
 
-The `target` directory is created as part of your initial project build, triggered by DevAssistant command above. This directory is where your compiled artifacts, or generated outputs (from builds) are stored. Its best to think of this directory as a temp space, used to store the outputs from a build. 
+This project folder is not a Git repository yet. Create a git repo:
+```
+$ cd my-app
+$ git init
```

Comment:
I plan to do a standalone `git` quickstart, so I'll move most of this later on, leaving just a reference.